maro/simulator/scenarios/cim/business_engine.py

```diff
@@ -662,11 +666,9 @@ def _on_action_received(self, event: CascadeEvent):
             event (CascadeEvent): Action event object with expected payload: {vessel_id: empty_number_to_move}}.
         """
         actions = event.payload
+        assert isinstance(actions, Iterable)
```

Comment:
Why `Iterable` not `List` here?

Reply:
I wrote this to maintain better flexibility (for example, the `actions` might be a `deque()`). But it is OK to use `List` here for now. I will change that.